build/release/teamcity-publish-release.sh

```diff
@@ -162,3 +162,49 @@ else
   echo "The ${dockerhub_repository}:latest docker image tag was _not_ pushed."
 fi
 tc_end_block "Tag docker image as latest"
+
+
+tc_start_block "Verify docker images"
+
+images=(
+  "${dockerhub_repository}:${build_name}"
+  "${gcr_repository}:${build_name}"
+)
+if [[ -z "$PRE_RELEASE" ]]; then
+  images+=("${dockerhub_repository}:latest-${release_branch}")
+fi
+if [[ -n "${PUBLISH_LATEST}" ]]; then
+  images+=("${dockerhub_repository}:latest")
+fi
+
+errorred=0
+
+for img in "${images[@]}"; do
+  docker rmi "$img"
+  docker pull "$img"
+  output=$(docker run "$img" version)
```

Comment:
Maybe check the output of `cockroach version --build-tag` too? I expect that will be more commonly used by scripts to check the version than greping for it in the full `version` output.

Reply:
Sweet, will add this check too!